test/db.tests.js

```diff
@@ -302,6 +307,24 @@ describe('LimitDBRedis', () => {
       });
     });
 
+    it('can parse a data and expire and override', (done) => {
```

Comment:
minor typo: "can parse a datE and expire and override"?

Reply:
👍